stories/IsolatedBlockEditor.stories.tsx

```diff
@@ -12,6 +18,62 @@ export const Default = () => {
 	return <IsolatedBlockEditor settings={ {} } />;
 };
 
+export const Managed = ( { onInput, onChange, onUndo, onRedo } ) => {
```

Comment:
Would "Controlled" be a more standard term for this?

Reply:
Yes, makes more sense I think. I'm renaming it. Thanks!